app/assets/javascripts/test-dependencies.js

```diff
@@ -3,4 +3,4 @@
 //= require govuk_publishing_components/dependencies
 //= require govuk_publishing_components/lib/cookie-functions
 //= require govuk_publishing_components/lib/header-navigation
-//= require govuk_publishing_components/lib/track-click
+//= require govuk_publishing_components/analytics
```

Comment:
Unless tests are failing, we shouldn't need this line, for the same reason mentioned above, should be available already via static. That being said, I'm not sure how the JavaScript tests are being set up in this repo, hence the question around failing tests 😬